README.asciidoc

```diff
@@ -0,0 +1,7 @@
+# Kivipallur-Jurto
```

Comment:
It's not recommended to change root directory README.asciidoc in any branches other than the documentation branch, otherwise there probably will be conflicts when merging.